report_pre_thesis/src/sections/methodology.tex

```diff
@@ -1,8 +1,191 @@
+
+Similarly, in our recreation of the pipeline, we have not incorporated outlier detection in the ICA part of the pipeline primarily because \citeauthor{cleggRecalibrationMarsScience2017} does not describe their implementation sufficiently enough for precise replication.
+They mention the use of Median Absolute Deviation for this purpose, but the specifics of their approach remain unclear.
+Our decision to omit outlier detection in the ICA phase of our pipeline is also influenced by a preference for retaining a more comprehensive dataset, despite the potential inclusion of outliers.
+Further supporting this decision is input from one of the original authors involved in the PLS1-SM aspect of the project.
+This author highlighted the extensive efforts invested in developing the ChemCam calibration dataset, suggesting a low probability of significant outlier presence.
+Given this context, introducing outlier detection for the ICA phase based on mere assumptions would be counterproductive, potentially compromising the integrity of our analysis.
+
+After examining the results in section \ref{sec:results}, we will discuss the implications of these design choices in section \ref{sec:discussion}.
+
+\subsection{Joint Approximate Diagonalization of Eigenmatrices (JADE) and Regression}
+After preprocessing the data, we are ready to perform ICA.
+This is done using the JADE algorithm, which is the same algorithm used by \citeauthor{cleggRecalibrationMarsScience2017}.
+This gives us eight independent components --- same as in the original pipeline.
+After performing ICA, we post-process by computing the correlation between the independent components and the wavelengths.
+Using this, we can identify which wavelengths are associated with which independent components by computing the maximum correlation for each wavelength.
+This gives us a matrix of ICA scores that we then use for regression.
+
+\begin{figure}
+	\centering
+	\includegraphics[width=0.45\textwidth]{images/ica_regression.png}
+	\caption{Regression in the ICA phase of our recreation of the pipeline.}
+	\label{fig:ica_regression}
+\end{figure}
+
+The regression process is illustrated in figure \ref{fig:ica_regression}.
+Since we are implementing this recreation in Python, a different programming language than the original pipeline, we cannot use the same libraries for regression.
+We use the \texttt{LinearRegression} class from the \texttt{scikit-learn} library\cite{scikit-learn}.
+However, \citeauthor{cleggRecalibrationMarsScience2017} mentions performing four more types of regression in addition to linear regression, namely parabolic, exponential, geometric and logsquare.
+For each oxide, they used the type of regression and normalization that produced the lowest root mean square (RMS) error.
+This can be seen in table \ref{tab:regression_types}, which is taken from \citeauthor{cleggRecalibrationMarsScience2017}.
+
+\begin{table}[h]
+\centering
```

Comment:
IMO don't duplicate their table. We could do it but only to also embed our own results, to showcase our baseline.